hacks/ghelp

```diff
@@ -79,13 +79,11 @@ if ghelp_info is None:
     exit(1)
 
 apttools = retrieve_packages_info(ghelp_info["apt"], "apt", "show", "=", "\n\n" , "Package", "Version", "Description")
-piptools = retrieve_packages_info(ghelp_info["pip"], "pip", "show", "==", "---", "Name", "Version", "Summary")
```

Comment:
Why did you remove this? Shouldn't we instead modify `retrieve_packages_info` to not do anything if the list of installed packages is empty?

P.S. there's still handling in the help generator for tools installed via pip: https://github.com/gardener/ops-toolbelt/blob/f04ab43a645098a45eb6a8768d6e8130b9f45e52/generator/lib/commands.py#L157